slimmer.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'null_logger'
   s.add_dependency 'rest-client'
   s.add_dependency 'activesupport'
+  s.add_dependency 'colorize'
```

Comment:
maybe, but it is good to have colored error messages, civil servants are users :D